frontend/src/components/homeview/homeview_components/DashboardStudyKing.jsx

```diff
@@ -11,16 +14,24 @@ class DashboardStudyKing extends Component {
       //[20, 23],
     ],
   };
+
+  componentDidMount() {
+    //todo: handle error properly
+    StudyKingService.getStudyKing()
+      .then((data) => this.setState(data))
+      .catch((msg) => console.log(msg));
```

Comment:
- [ ] if msg is error message then please use console.err and maybe alert if this message is important for the end user